package.json

```diff
@@ -92,7 +92,32 @@
         "language": "dart",
         "path": "./snippets/flutter.json"
       }
-    ]
+    ],
+    "configuration": {
+      "title": "Custom Wraps",
+      "properties": {
+        "flutterplus.customWraps": {
```

Comment:
That's a `flutter-plus`

Also, let's name it as a just `wraps`.

`flutter-plus.wraps`